.github/workflows/build-containers.yml

```diff
@@ -71,6 +77,7 @@ jobs:
               type=semver,pattern={{major}}
               type=ref,event=branch
               type=ref,event=pr
+              type=raw,value=latest,enable=${{ github.ref == format('refs/tags/{0}', steps.latest.outputs.tag) }}
```

Comment:
Also I want to make you aware of [another issue](https://github.com/docker/metadata-action/issues/461#issuecomment-2680849083) that is specific to `type=raw,value=latest,enable=false`.

That will have the action skip processing this tag line, but without changing the `flavor.latest=auto` default, any of the other tag types that implicitly enable the `latest` flag like the semver one you have will cause the `latest` tag to be updated too.